raspi/README.md

```diff
@@ -0,0 +1,232 @@
+**Upload** files from your computer to it. Click the **Bootstrap** button to
+install some programs (see [these details][pf2]).
+
+  [pif]: https://github.com/adafruit/Adafruit-Pi-Finder
+  [pf2]: https://learn.adafruit.com/the-adafruit-raspberry-pi-finder/overview
+
+Install our software project on the system
+------------------------------------------
+
+Using either the **Terminal** button on the *Pi Finder* application, or starting
+up the **Terminal** on your laptop and typing:
+
+    $ ssh pi@10.0.1.187  # Or whatever the IP address is
+
+Once you are logged into the system, let's first secure the system by changing the password:
+
+    $ passwd
+    Changing password for pi.
+    (current) UNIX password:       # This is probably raspberry
+    Enter new UNIX password:       # Set to pigmice2733
+    Retype new UNIX password:
+    passwd: password updated successfully
+
+Next, let's install `git` and the other dependencies:
+
+    $ sudo apt-get install -y git python-pip python-opencv
+    $ sudo apt-get install -y python-numpy python-yaml
+    $ sudo pip install pynetworktables
+
+Now, we can get the latest version of our code on to the system:
+
+    $ git clone https://github.com/pigmice2733/pigmice-vision
+
+Once you `cd` into the project, test it out:
+
+    $ cd pigmice-vision
+    $ pip install -r requirements.txt
+
+Remember, you will need to have a valid configuration file, so you may want to
+start with the one on your laptop. To copy this, go to the **Terminal** on your
+laptop, and enter:
+
+    $ scp .pigmice-config.yaml pi@10.0.1.187:
+
+Static IP Address
+------------------
+We will be setting the static (not changing) ip address through the pi's dhcp
+file. To change this, nano (this allows us to edit a file through the command
+line) into the file.
+After ssh-ing into the pi, type this command in the command prompt:
+    $ sudo nano /etc/dhcpcd.conf
+
+Scroll to the bottom and un-comment this line
+    $ static ip_address=192.168.1.23/24
+
+Change the IP address to start with the first three numbers of your computer
+(You can find your computer's IP address by typing ifconfig).
+For example the roborio's IP address starts with 10.27.33
+    $ static ip_address=10.27.33.23/24
+
+The last number can be whatever you'd like.
+    $ static ip_addres=10.27.33.20
+
+Setting up the files in the raspi folder
+----------------------------------------
+
+After downloading the raspi folder from github you'll need to move some of the
+files to different directories on your computer.
+Move requirements.text and pigmice-vision.service to your home directory.
+Requirements.text will be run on the raspi to download all the necessary
+libraries and packages.
+
+If you want to run the program on your computer you'll need to make a copy of
+it and change the routes, for example:
+Instead of WorkingDirectory=/home/pi/pigmice-vision
+you would instead write WorkingDirectory=/home/user/github/pigmice-vision
```

Comment:
While this is all true, I don't think any one will want to run this on their laptop as a service (as they can just run the program directly), so maybe we shouldn't confuse the issue?